src/map/script.cpp

```diff
@@ -13244,26 +13244,50 @@ BUILDIN_FUNC(getwaitingroomstate)
 	if( script_hasdata(st,3) )
```

Comment:
The in-source function documentation needs an update.
Maybe most of it should be moved into the enum?